src/test/java/org/broadinstitute/hellbender/engine/spark/AddContextDataToReadSparkUnitTest.java

```diff
@@ -70,7 +70,7 @@ public void addContextDataTest(List<GATKRead> reads, List<GATKVariant> variantLi
         SAMSequenceDictionary sd = new SAMSequenceDictionary(Lists.newArrayList(new SAMSequenceRecord("1", 100000), new SAMSequenceRecord("2", 100000)));
         when(mockSource.getReferenceSequenceDictionary(null)).thenReturn(sd);
 
-        JavaPairRDD<GATKRead, ReadContextData> rddActual = AddContextDataToReadSpark.add(ctx, rddReads, mockSource, rddVariants, joinStrategy,
+        JavaPairRDD<GATKRead, ReadContextData> rddActual = AddContextDataToReadSpark.add(ctx, rddReads, mockSource, rddVariants, null, joinStrategy,
```

Comment:
I tried to do this, but I couldn't see an easy way to adapt the test to write the collection of variants to a file since constructing a valid VCF header was not obvious. However, this code path is tested by BQSRPipelineSparkIntegrationTest, so it should be OK.